nac_collector/controller/meraki_dashboard_api/__init__.py

```diff
@@ -0,0 +1,2 @@
+__version__ = '2.0.3'
+__api_version__ = '1.58.0'
```

Comment:
`meraki_dashboard_api` is copied from part of `dashboard-api-python` (as mentioned in the commit message and PR description). The only reason for copying it rather than importing it was to avoid pulling in extra dependencies which are not used by the parts of the library we use.
I also changed this copy to use `httpx` instead of `requests` to avoid pulling that in.

So importing it instead would be fine if you prefer that.

@danischm you mentioned a third option - extending the existing nac-collector client to return errors - but that proved difficult for me.

Reply:
I've removed the commits that copied the library - now it is imported normally and pulls in dependencies in `pyproject.toml`.